x-pack/plugins/monitoring/server/kibana_monitoring/classes/collector_set.js

```diff
@@ -139,6 +137,19 @@ export class CollectorSet {
     });
   }
 
+  async bulkFetchUsage() {
```

Comment:
Note that this isn't used yet. See https://github.com/elastic/kibana/pull/18894 for where it'll be used from an HTTP API

cc @ruflin the API will call this function from the handler, meaning that the stats will get collected whenever the API is called. No internal caching